src/View/Form/ContextInterface.php

```diff
@@ -70,6 +70,14 @@ public function val($field);
      */
     public function isRequired($field);
 
+    /**
+     * Gets the default "required" error message for a field
+     *
+     * @param string $field A dot separated path to the field
+     * @return string
+     */
+    public function getRequiredMessage($field);
```

Comment:
Changing interfaces usually has to wait until a major version as it can break other implementations. I know that the elastic search plugin also uses this interface.

Reply:
Since we don't have a 4.0 branch yet, should I just close this? Or let it sit and re-target after there is one?

Reply:
Can't we annotate the interface method as doc block as done in other places for 3.x?

Reply:
So far we've been annotating interfaces and using `method_exists` as an interim solution.